data_validation/cli_tools.py

```diff
@@ -1062,8 +1062,16 @@ def get_validation(validation_name, config_dir=None):
         mgr = state_manager.StateManager(file_system_root_path=config_dir)
         return mgr.get_validation_config(validation_name, config_dir)
     else:
-        mgr = state_manager.StateManager()
-        return mgr.get_validation_config(validation_name)
+        if validation_name.startswith("gs://"):
+            obj_depth = len(validation_name.split("/"))
+            gcs_prefix = "/".join(validation_name.split("/")[: obj_depth - 1])
```

Comment:
This might be easier than getting the depth - 1: `"/".join(file.split("/")[:-1])`